modules/installation-creating-aws-vpc.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_upi/installing-aws-upi.adoc
+
+[id="installation-creating-aws-vpc-{context}"]
+= Creating a VPC in AWS
+
+You must create a VPC in Amazon Web Services (AWS) for your {product-title}
+cluster to use. You can customize the VPC to meet your requirements, including
+VPN and route tables. The easiest way to create the VPC is to modify the
+provided CloudFormation template.
+
+[NOTE]
+====
+If you do not use the provided CloudFormation template to create your AWS
+infrastructure, you must review the provided information and manually create
+the infrastructure. If your cluster does not initialize correctly, you might
+have to contact Red Hat support with your installation logs.
+====
+
+.Prerequisites
+
+* Configure an AWS account.
+* Generate the Ignition config files for your cluster.
+
+.Procedure
+
+. Create a YAML file that contains a copy of the CloudFormation template that
+describes the VPC and save it to your computer.
+
+. Create a JSON file that contains the parameter values that the template
+requires:
++
+[source,json]
+----
+[
+  {
+    "ParameterKey": "VpcCidr", <1>
+    "ParameterValue": "10.0.0.0/16" <2>
+  },
+  {
+    "ParameterKey": "AvailabilityZoneCount", <3>
+    "ParameterValue": "1" <4>
+  },
+  {
+    "ParameterKey": "SubnetBits", <5>
+    "ParameterValue": "12" <6>
+  }
+]
+----
+<1> The CIDR block for the VPC.
+<2> Specify a CIDR block in the format `x.x.x.x/16-24`.
+<3> The number of availability zones to deploy the VPC in.
+<4> Specify an integer between `1` and `3`.
+<5> The size of each subnet in each availability zone.
+<6> Specify an integer between  `5` and `13`, where `5` is `/27` and `13` is `/19`.
+
+. Launch the template:
++
+[IMPORTANT]
+====
+You must enter the command on a single line.
+====
++
+----
+$ aws cloudformation create-stack --stack-name <name> <1>
+     --template-body file://<template>.yaml <2>
+     --parameters file:///<parameters>.json <3>
```

Comment:
The prior template-body  file location uses // for present working directory.  The <parameters>.json file should follow suit.